styles/styles.css

```diff
@@ -0,0 +1,60 @@
+@import url('styles/fonts.css');
+
+.advantages {
+    width: 1440px;
+    margin: 0 auto;
+    padding: 120px 16px 120px;
+}
+
+.title {
+    text-align: center;
+    font-family: 'Roboto medium', sans-serif;
+    font-size: 40px;
+}
+
+.line {
+    max-width: 1440px;
+    width: 100px;
+    height: 4px;
+    background: #f5bd41;
+    margin: 20px auto 85px;
+}
+
+.row {
+    display: flex;
+    gap: 16px;
+    justify-content: center;
+}
+
+.col {
+    width: 300px;
+    height: 180px;
+    text-align: center;
+    font-family: 'Roboto regular', sans-serif;
+    font-size: 24px;
+}
+
+.footer {
+    width: 1440px;
+
+    margin: 0 auto;
+    background: #10367C;
+}
+
+.menu-row {
+    display: flex;
+    gap: 0 260px;
+    padding: 120px 0 120px 105px;
+    flex-wrap: wrap;
+}
```

Comment:
You can add width:33.33333% in menu-col class and aligned it more flexible using flexes properties